sound/soc/sof/topology.c

```diff
@@ -1018,7 +1007,7 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	/* set up possible led control from mixer private data */
 	ret = sof_parse_tokens(scomp, &scontrol->led_ctl, led_tokens,
 			       ARRAY_SIZE(led_tokens), mc->priv.array,
-			       le32_to_cpu(mc->priv.size));
+			       le32_to_cpu(mc->priv.size), 1, 0);
```

Comment:
you could probably reduce the volume of patches by introducing a helper function, e.g.

sof_parse_token_array()

sof_parse_tokens
    sof_parse_token_array(1, 0)

also move the changes in tests to a separate patch.

Reply:
ok good idea, thanks.